contracts/bridge/Bridge.sol

```diff
@@ -99,10 +112,15 @@ contract Bridge is ITokenReceiver, INFTReceiver, Ownable {
     {
         require(handleNonce == _requestNonce, "mismatched handle / request nonce");
 
-        emit HandleValueTransfer(_to, TokenKind.TOKEN, _contractAddress, _amount, handleNonce);
+        emit HandleValueTransfer(_to, TokenKind.ERC20, _contractAddress, _amount, handleNonce);
         lastHandledRequestBlockNumber = _requestBlockNumber;
         handleNonce++;
-        IERC20(_contractAddress).transfer(_to, _amount);
+
+        if (modeMintBurn) {
```

Comment:
I think we should actually separate it into parent/child, so why don't we change the variable name to `modeMintBurn --> isParent`?

Reply:
I think SP can want mintBurnMode for parent also. So I didn't name it like isChild.